TreeDB/documentservice/http_test.go

```
@@ -66,6 +66,56 @@ func TestHTTPMalformedJSONKeywordHybridAndErrorPayloads(t *testing.T) {
 	assertHTTPErrorCode(t, rr.Body.Bytes(), CodeInvalidRequest)
 }
 
+func TestHTTPHybridParentCollapseRequestResponse4291(t *testing.T) {
+	svc, db := newTestService(t)
+	defer db.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)